From 5988b61f07d044fa59f61216e6369dc45d077c0a Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Fri, 28 Jun 2024 08:46:13 +1100
Subject: [PATCH] MAINT: fix typo in small_dynamic_array.h

on clang-19, this causes:
```
../scipy/_lib/_uarray/small_dynamic_array.h(145,18): error: reference to non-static member function must be called
  145 |     size_ = copy.size;
      |             ~~~~~^~~~
1 error generated.
```
---
 scipy/_lib/_uarray/small_dynamic_array.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scipy/_lib/_uarray/small_dynamic_array.h b/scipy/_lib/_uarray/small_dynamic_array.h
index b6c46d7c44fa..351b5d8fc653 100644
--- a/scipy/_lib/_uarray/small_dynamic_array.h
+++ b/scipy/_lib/_uarray/small_dynamic_array.h
@@ -142,7 +142,7 @@ class SmallDynamicArray {
 
     clear();
 
-    size_ = copy.size;
+    size_ = copy.size_;
     try {
       allocate();
     } catch (...) {
